### PR TITLE
Event module API calling

### PR DESCRIPTION
**Pull request by ShrutiSachpara**

### CODE REVIEW

File: src/app/manage-event/manage-event.component.ts [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+    this.currentPage = page;
+  }
+
+  showMessage(message: string, action: string) {
+    this.toaster.showMessage(message, action);
+  }
+
+  listOfEvent() {
+    this.manageEventService.getEventData().subscribe((response: any) => {
+      if (response && response.status === 'success') {
```
Comment (krishna2919) [inline src/app/manage-event/manage-event.component.ts]: please use enum for success

File: src/app/base/base-table/base-table.component.html [hunk 2]
```diff
@@ -11,18 +11,50 @@
...
+            sortedColumn === column && isAscending
+              ? "arrow_upward"
+              : "arrow_downward"
+          }}
+        </mat-icon>
+      </span>
     </th>
-    <td mat-cell *matCellDef="let item">{{ getColumnValue(item, column) }}</td>
+    <td mat-cell *matCellDef="let item; let j = index">
+      <ng-container *ngIf="column === 'Sr. No'">
```
Comment (SwetaaRampariya) [inline src/app/base/base-table/base-table.component.html]: please remove unnecessary ng-if's as we already discussed

File: src/app/base/base-table/base-table.component.ts [hunk 3]
```diff
@@ -25,7 +25,11 @@ export class BaseTableComponent implements OnChanges {
   @Input() columns: string[] = [];
   @Input() sortedColumn: string = '';
   @Input() isAscending: boolean = true;
+  @Input() columnActionTemplate: any;
   @Output() pageChanged: EventEmitter<number> = new EventEmitter<number>();
+  @Output() onEdit = new EventEmitter<any>();
+  @Output() onDelete = new EventEmitter<any>();
+
   totalItemsCount: number = 0;
   displayedData: any[] = [];
```
Comment (SwetaaRampariya) [inline src/app/base/base-table/base-table.component.ts]: Do not use any

File: src/app/base/base-table/base-table.component.ts [hunk 4]
```diff
@@ -60,6 +64,12 @@ export class BaseTableComponent implements OnChanges {
     const filterValue = (event.target as HTMLInputElement).value;
     this.searchSubject.next(filterValue);
   }
+  editRecord(item: any) {
```
Comment (SwetaaRampariya) [inline src/app/base/base-table/base-table.component.ts]: do not use any type.

File: src/app/manage-event/manage-event.component.ts [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+    if (selectedEvent) {
+      this.isEditMode = true;
+      this.selectedEventId = data.id;
+      this.GetNewEvent.patchValue(selectedEvent);
+      this.openModal();
+    }
+  }
+
+  deleteEvent(data: any) {
+    Swal.fire({
```
Comment (SwetaaRampariya) [inline src/app/manage-event/manage-event.component.ts]: make comman file for toast message

**Pull request merged**